package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "mybrand-api",
+  "version": "1.0.0",
+  "description": "this is a backend version of my brand",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha test/project-setup-test.js",
+    "dev-start": "babel-watch server.js",
+    "build": "babel server.js --out-dir build",
+    "start": "babel-node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pextech/mybrand-api.git"
+  },
+  "author": "pextech",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/pextech/mybrand-api/issues"
+  },
+  "homepage": "https://github.com/pextech/mybrand-api#readme",
+  "dependencies": {
+    "@babel/cli": "^7.12.8",
+    "@babel/core": "^7.12.9",
+    "@babel/node": "^7.12.6",
+    "@babel/preset-env": "^7.12.7",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.2.2",
+    "body-parser": "^1.19.0",
```

Comment:
Remove babel-loader bcz you won't be needing it as well as body-parser because you are using `express.json()` to accept JSON a body from your requests.